### ISSUE TITLE
Fps drops on CopyChat

### ISSUE THREAD

**Issue by Ssesmar** [Legacy-of-Sylvanaar/prat-3-0#92]
Hi, every time i use the copychat, i got hard fps drops from 117 fps to around 30-40fps, when i close the window the fps are back to normal.
And copychat highlight doest works.

**Comment by plusmouse** [comment]
Does this happen regardless of how much text is available in the text box?

**Comment by Ssesmar** [comment]
no, it only happens on the general channel. no highlight and fps drops. For example, if I want to copy something from the combat log, there are no fps drops and the highlit works too.
